messages/dji_msgs/msg/Topics.msg

```diff
@@ -1,9 +1,30 @@
 # Used by the dji_captain, namespaced as needed.
 # Usually under /M350
-string MOVE_TO_SETPOINT_TOPIC = 'move_to_setpoint'
-string CONTROLLER_INPUT_TOPIC = 'joy'
-string CONTROLLER_VIBRATION_TOPIC = 'joy/set_feedback'
-string TTS_TOPIC = 'speak'
 
 
+string MOVE_TO_SETPOINT_TOPIC = 'move_to_setpoint' # PoseStamped
 
+# user feedback
+string CONTROLLER_INPUT_TOPIC = 'joy' # Joy
+string CONTROLLER_VIBRATION_TOPIC = 'joy/set_feedback' # JoyFeedback
+string TTS_TOPIC = 'speak' # String
+
+# gimbal camera
+string GIMBAL_CAMERA_RAW_TOPIC = 'gimbal_camera/image_raw' # Image
+
+# object detectors, these are in normalized camera pixel coordinates
+string ESTIMATED_BUOY_TOPIC = 'alars_detection/buoy' # PointStamped
+string ESTIMATED_AUV_TOPIC = 'alars_detection/auv' # PointStamped
+string ESTIMATED_MIDDLE_TOPIC = 'alars_detection/middle' # PointStamped
+
+# services
+string ENABLE_ALARS_DETECTOR_SERVICE_TOPIC = 'enable_alars_detector' # Trigger
+string DISABLE_ALARS_DETECTOR_SERVICE_TOPIC = 'disable_alars_detector' # Trigger
+
+# sensors that are not coming from PSDK
+string LOAD_CELL_WEIGHT_TOPIC = 'sensor/load_cell_weight' # Float32
+
+# Used by hook_estimator
+string CAMERA_TOPIC = 'core/fpcamera/image'
+string HOOK_ESTIMATE_TOPIC = 'hook_estimate'
+string HOOK_EKF_ESTIMATE_TOPIC = 'hook_ekf_estimate'
```

Comment:
please match the other topics
`ESTIMATED_HOOK_TOPIC = 'alars_detection/hook' #PoseStamped` 
(PoseStamped is the type you'll be publishing, might be sth else)